### PR TITLE
edit cmd/kubelet/app/plugins_providers.go to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/kubelet/app/plugins_providers.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/kubelet/app/plugins_providers.go [hunk 1]
```diff
@@ -41,11 +41,11 @@ func appendPluginBasedOnMigrationFeatureFlags(plugins []volume.VolumePlugin, inT
 	// CSI migration is complete
 	err := csimigration.CheckMigrationFeatureFlags(featureGate, pluginMigration, pluginMigrationComplete)
 	if err != nil {
-		klog.Warningf("Unexpected CSI Migration Feature Flags combination detected: %v. CSI Migration may not take effect", err)
+		klog.InfoS("Unexpected CSI Migration Feature Flags combination detected: %v. CSI Migration may not take effect", err)
```
Comment (kikisdeliveryservice) [inline cmd/kubelet/app/plugins_providers.go:44]: are we intending on changing from warning to info?

**Comment by kikisdeliveryservice** [comment]
@ehashman this seems like incorrect label should be kind/cleanup not kind/documentation no?

/ok-to-test

File: cmd/kubelet/app/plugins_providers.go (hunk 1 above)
Comment (kikisdeliveryservice) [reply cmd/kubelet/app/plugins_providers.go:44]: ah according to: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent the answer is yes :+1:

File: cmd/kubelet/app/plugins_providers.go [hunk 2]
```diff
@@ -41,11 +41,11 @@ func appendPluginBasedOnMigrationFeatureFlags(plugins []volume.VolumePlugin, inT
...
 		// TODO: fail and return here once alpha only tests can set the feature flags for a plugin correctly
 	}
 	if featureGate.Enabled(features.CSIMigration) && featureGate.Enabled(pluginMigration) && featureGate.Enabled(pluginMigrationComplete) {
-		klog.Infof("Skip registration of plugin %s since feature flag %v is enabled", inTreePluginName, pluginMigrationComplete)
+		klog.InfoS("Skip registration of plugin %s since feature flag %v is enabled", inTreePluginName, pluginMigrationComplete)
```
Comment (kikisdeliveryservice) [inline cmd/kubelet/app/plugins_providers.go:48]: as per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message this needs to be reconfigured

**Review by kikisdeliveryservice — CHANGES_REQUESTED** [review]
Please see: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message specifically: "All string formatting (%d, %v, %w, %s) should be removed and log message string simplified."

This means that the changes in this PR are incomplete and the messages need to be updated as per the guidelines above.

**Comment by kikisdeliveryservice** [comment]
As this is a user facing change adding release-note to be safe, please feel free to modify if this is incorrect

/release-note

File: cmd/kubelet/app/plugins_providers.go (hunk 1 above)
Comment (serathius) [inline cmd/kubelet/app/plugins_providers.go:44]: Printf syntax like `%v` is not supported by `InfoS` method.
Please check out documentation about updating messages and removing formatting https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

**Pull request closed without merging**